INN/umbrella-borderzine
# Add subscribe button to global and sticky navs

## Changes

This pull request makes the following changes:

- Adds a "Subscribe" button to the global nav and sticky nav next to the "Donate" button 

(ignore the missing icon font in these screenshots)
Desktop:
![Screen Shot 2020-05-04 at 1 29 58 PM](https://user-images.githubusercontent.com/18353636/80995015-9099bb80-8e0b-11ea-8000-3c8e04c448f5.png)

Sticky:
![Screen Shot 2020-05-04 at 1 30 05 PM](https://user-images.githubusercontent.com/18353636/80995016-91325200-8e0b-11ea-8afa-0bc349ba19ea.png)

Mobile:
![Screen Shot 2020-05-04 at 1 31 17 PM](https://user-images.githubusercontent.com/18353636/80995019-91cae880-8e0b-11ea-80d7-68b1aa1fce35.png)

## Why



For #83 / Giving Tuesday

## Testing/Questions

Features that this PR affects:

- Global nav, sticky nav, mobile nav


Questions that need to be answered before merging:

- [ ] Are the colors ok?
- [ ] The donate button has an icon inside of it. Does the subscribe button need an icon as well?
- [ ] The donate button is hidden on the mobile menu. Should the subscribe button be hidden as well?

Steps to test this PR:

1. View the subscribe button on the global nav on desktop
2. View the subscribe button on the sticky nav on desktop
3. View the subscribe button on the mobile nav
